docs/content/topics/scripting/guide.md

```diff
@@ -9,1401 +9,867 @@ aliases:
 
 ```javascript
-const { events } = require("brigadier")
+const { events } = require("@brigadecore/brigadier");
 
-events.on("exec", () => {
-  console.log("==> handling an 'exec' event")
-})
-```
-[brigade-02.js](../../examples/brigade-02.js)
+events.on("brigade.sh/cli", "exec", async event => {
+  console.log("Hello, World!");
 
-> The `() => {}` syntax is a newer way to declare an anonymous function. We use this
-> syntax throughout the tutorial, but unless otherwise noted, you can substitute in the
-> traditional syntax: `function () {}`.
+  // Optionally return a string and it will automatically be persisted
+  // in the event's summary field.
+  return "All done."
+});
+
+events.process();
+```
+[02-first-event](https://github.com/brigadecore/brigade/tree/v2/examples/02-first-event)
 
 There are a few things to note about this script:
 
-- It imports the `events` object from `brigadier`. Almost all Brigade scripts do this.
-- It declares a single event handler that says "on an 'exec' event, run this function".
+- It imports the `events` object from `@brigadecore/brigadier`. Almost all
+  Brigade scripts do this.
+- It declares a single event handler that says "on an 'exec' event, run this
+  function".
+- We're not actually utilizing the `event` object in this script, but we will
+  in a later example
+- The event handler returns a string, which is the [event summary]. This is can
+  be utilized by gateways interested in the results or other context related
+  to the handling of the event.
+- All scripts must call `events.process()` in order to start up the node
+  process that the script will run in.
+
+Similarly to our first script, this event handler function displays a message
+to a log, producing the following output:
 
-Similarly to our first script, this function simply dumps a message to a log.
+```console
+$ brig event create --project first-event --follow
 
-The above produces something like this:
+Created event "5b0bd00a-4f31-40da-ad01-0d2f62f4d70e".
 
-```console
-Started brigade-worker-01brx1z21af78hsj4q55anycpc-master
-yarn start v0.27.5
-$ node prestart.js
-prestart: src/brigade.js written
-$ node --no-deprecation ./dist/src/index.js
-Creating PVC named brigade-worker-01brx1z21af78hsj4q55anycpc-master
-==> handling an 'exec' event
-Destroying PVC named brigade-worker-01brx1z21af78hsj4q55anycpc-master
-Done in 1.49s.
+Waiting for event's worker to be RUNNING...
+2021-09-20T22:24:57.655Z INFO: brigade-worker version: a398ba8-dirty
+Hello, World!
 ```
 
-One of the strengths of Brigade is that we can handle different events in the same file.
-Brigade will only trigger the appropriate events. For example, we can expand the example
-above to also provide a handler for the GitHub `push` event:
+One of the strengths of Brigade is that we can handle different events in the
+same file. Brigade will only trigger the appropriate events. For example, we
+can expand the example above to also provide a handler for a GitHub push event.
```

Comment:
Rather than assert this is a strength, can we maybe connect the dots more explicitly here. Hopefully the readers will conclude on their own that it is a strength.

Could we say that since projects may subscribe to multiple types of events (from multiple sources, too), defining multiple event handlers in your script permits different types of events to be _handled_ differently?